exasol/sagemaker_extension_wrapper.py

```diff
@@ -0,0 +1,149 @@
+from exasol_sagemaker_extension.deployment.deploy_create_statements import DeployCreateStatements   # type: ignore
+from exasol_sagemaker_extension.deployment.language_container_deployer import LanguageActivationLevel   # type: ignore
+from exasol_sagemaker_extension.deployment.sme_language_container_deployer import SmeLanguageContainerDeployer  # type: ignore
+
+from exasol.connections import (
+    get_external_host,
+    open_pyexasol_connection,
+)
+from exasol.extension_wrapper_common import (
+    encapsulate_aws_credentials,
+    str_to_bool
+)
+from exasol.language_container_activation import (
+    ACTIVATION_KEY_PREFIX,
+    get_activation_sql
+)
+from exasol.secret_store import Secrets
+
+# Root directory in a bucket-fs bucket where all stuff of the Sagemaker
+# Extension, including its language container, will be uploaded.
+PATH_IN_BUCKET = "SME"
+
+LATEST_KNOW_VERSION = "0.6.0"
```

Comment:
```suggestion
LATEST_KNOWN_VERSION = "0.6.0"
```

Reply:
Ah, this type is reemerging like a phoenix bird. I thought I fixed it. Sorry, and thanks.